packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -213,7 +213,8 @@ export default function CodeBlock({
               tabIndex={0}
               className={clsx(className, styles.codeBlock, 'thin-scrollbar', {
                 [styles.codeBlockWithTitle]: codeBlockTitle,
-              })}>
+              })}
+              style={style}>
               <div className={styles.codeBlockLines} style={style}>
```

Comment:
I think `style={style}` can be removed from this line as it is there in parent now. @lex111 @slorber 
What do you think?

Reply:
I don't understand what you mean